spec/index_spec.rb

```diff
@@ -76,7 +76,7 @@
   context "#[]" do
     before do
       @id = Daru::Index.new [:one, :two, :three, :four, :five, :six, :seven]
-      @mixed_id = Daru::Index.new ['a','b','c',:d,:a,0,3,5]
+      @mixed_id = Daru::Index.new ['a','b','c',:d,:a,8,3,5]
```

Comment:
Alright. But keep in mind that if `0` exists in the named Index, the Index must return the corresponding element position.